src/engine/outfit.ts

```diff
@@ -395,8 +395,14 @@ export function equipDefaults(outfit: Outfit, noFightingFamiliars: boolean): voi
       outfit.equip($item`fresh coat of paint`);
     }
     outfit.equip($item`familiar scrapbook`);
-    outfit.equip($item`protonic accelerator pack`);
-    outfit.equip($item`unwrapped knock-off retro superhero cape`);
+    // eslint-disable-next-line libram/verify-constants
+    if(have($item`bat wings`) && get("_batWingsFreeFights",0) < 5) {
```

Comment:
Linter needs to be run on this, also I'm not sure if we want to prioritize this over the ghostbuster thingy if the ghostbuster thingy is queued. But I don't follow the logic.

Basically my line of thought is "do we really need to use all bat fights asap?"

Reply:
It didn't exist in lint yet!

These are all free and t0